libraries/abstractions/pkcs11/test/iot_test_pkcs11.c

```diff
@@ -344,7 +344,7 @@ static MultithreadTaskParams_t xGlobalTaskParams[ pkcs11testMULTI_THREAD_TASK_CO
 /*-----------------------------------------------------------*/
 /* Stack size of each task. This can be configured in iot_test_pkcs11_config.h. */
 #ifndef pkcs11testMULTI_TASK_STACK_SIZE
-    #define pkcs11testMULTI_TASK_STACK_SIZE    ( configMINIMAL_STACK_SIZE * 4 )
+    #define pkcs11testMULTI_TASK_STACK_SIZE    ( configMINIMAL_STACK_SIZE * 8 )
```

Comment:
This is a huge delta. Was this failing only on ESP?

Reply:
Yes, I believe this was passing on Windows.

Reply:
I feel like we should change this to be a number rather than a multiple of configMINIMAL_STACK_SIZE, since that varies so much platform to platform.   Could we use what you learned about the stack requirements on esp and pick an appropriate value here?

Reply:
@dcgaws suggested I move this to a config file. I agree I can move this to iot_test_pkcs11_config.h.